### ISSUE TITLE
Add content type-to-extension conversion

### ISSUE THREAD

**Issue by tmannherz** [dart-lang/mime#13]
Currently, the package only does file-to-content type conversion via the `lookupMimeType` function. As the extension maps already exist, it'd be easy to add both sides of the lookup, something like:

```dart
print(lookupExtension('text/html'));  // html
print(lookupExtension('application/pdf'));  // pdf
```

**Comment by kevmoo** [comment]
Pull requests welcome!

**Comment by d-silveira** [comment]
doesn't these reference commits close this issue?
@tmannherz did you request a pull?

**Comment by lifenautjoe** [comment]
Any clue when this might be released? 🙏

**Comment by ZuSe** [comment]
Pull request needs to be merged first
https://github.com/dart-lang/mime/pull/15

**Comment by kevmoo** [comment]
It was abandoned by the author. If someone else wants that functionality, starting with a new PR would be great!

**Comment by tmannherz** [comment]
Sorry, the related PR lingered for so long awaiting approval (~2 years) that I can't support it and don't do any active Dart development, so I closed it. Feel free to copy and take it over.
